examples/todo/card.rs

```diff
@@ -0,0 +1,40 @@
+use kayak_core::{
+    rsx,
+    styles::{LayoutType, Style, StyleProp, Units},
+    widget, Color, EventType, Handler, OnEvent,
+};
+use kayak_widgets::{Background, Text};
+
+use super::delete_button::DeleteButton;
+
+#[widget]
+pub fn Card(card_id: usize, name: String, on_delete: Handler<usize>) {
+    let name = name.clone();
+    let background_styles = Style {
+        layout_type: StyleProp::Value(LayoutType::Row),
+        background_color: StyleProp::Value(Color::new(0.176, 0.196, 0.215, 1.0)),
+        height: StyleProp::Value(Units::Pixels(26.0)),
+        top: StyleProp::Value(Units::Pixels(10.0)),
+        padding_left: StyleProp::Value(Units::Pixels(5.0)),
+        padding_right: StyleProp::Value(Units::Pixels(5.0)),
+        padding_top: StyleProp::Value(Units::Pixels(5.0)),
+        padding_bottom: StyleProp::Value(Units::Pixels(5.0)),
+        ..Style::default()
+    };
+
+    let on_delete = on_delete.clone();
```

Comment:
There's so much clone everywhere. I'm thinking in the [widget] proc macro we clone all props by default? Except maybe styles.. 🤔

Reply:
Makes sense. Plus if all props become a `Binding`, this shouldn't be that much of an issue anyways